build.proj

```diff
@@ -28,7 +28,7 @@
 
   <Import Project="Directory.Build.targets" />
 
-  <Target Name="Restore" DependsOnTargets="GenerateConfigurationProps;RestoreOptionalToolingPackages;RestoreOptimizationDataPackage">
+  <Target Name="Restore" DependsOnTargets="GenerateConfigurationProps;RestoreOptimizationDataPackage">
```

Comment:
Does this need to be fixed up/changed?

[https://github.com/dotnet/corefx/blob/e0cb81d4df9e6279cff5be3659750f1140d977e0/eng/crossIBC.ps1#L68](https://github.com/dotnet/corefx/blob/e0cb81d4df9e6279cff5be3659750f1140d977e0/eng/crossIBC.ps1#L68)

Reply:
No that still works